weTranslate/SearchViewController.swift

```diff
@@ -147,8 +148,17 @@ extension SearchViewController: UITableViewDelegate {
     }
 }
 
+
+extension SearchViewController: UIScrollViewDelegate {
+
+    func scrollViewWillBeginDragging(scrollView: UIScrollView) {
+        searchHeaderViewController.resignFirstResponder()
```

Comment:
Didn't want to expose a mutable state, `resignFirstResponder` seams like the natural method to call, it actually set the state to `.Default` [here](https://github.com/jenniferlim/weTranslate/pull/20/files#diff-1fdce09f8d3feaa1cd1cd3af41345741R97).